HDF4Examples/C/AN/C_sourcefiles.cmake

```diff
@@ -2,11 +2,11 @@
 # Define Sources, one file per application
 #-----------------------------------------------------------------------------
 set (h4examples
-    h4ex_AN_create_annotation.c
+    AN_create_annotation
 )
 set (h4examples_use
-    h4ex_AN_get_annotation_info.c
-    h4ex_AN_read_annotation.c
+    AN_get_annotation_info
+    AN_read_annotation
```

Comment:
These don't need some sort of H4/h4 prefix?

Reply:
No, that gets added in the CMakeLists.txt, simplified it.